features/bootstrap/FeatureContext.php

```diff
@@ -58,6 +81,10 @@ public function __construct(Request $request, ManagerRegistry $doctrine, Kernel
         $this->manager = $doctrine->getManager();
         $this->schemaTool = new SchemaTool($this->manager);
         $this->classes = $this->manager->getMetadataFactory()->getAllMetadata();
+        $this->accountsHelper = new AccountHelper($request);
```

Comment:
To get things cleaner, I would use an array, as we may have a lot of helpers later.

I would create an array:

```php
$this->helpers[AccountHelper::class] = new AccountHelper($request);
```